exoctk/contam_visibility/sossContamFig.py

```diff
@@ -60,31 +58,184 @@ def contam(cube, instrument, targetName='noName', paRange=[0, 360],
     rows = cube.shape[1]
     cols = cube.shape[2]
     print('cols ', cols)
-    dPA = 360//nPA
-    PA = np.arange(nPA)*dPA
+    dPA = 360 // nPA
+    PA = np.arange(nPA) * dPA
 
     contamO1 = np.zeros([rows, nPA])
-    if instrument=='NIRISS':
-        contamO2 = np.zeros([rows, nPA])
+    contamO2 = np.zeros([rows, nPA])
+
+    low_lim_col = 20
+    high_lim_col = 41
 
     for row in np.arange(rows):
+        # Contamination for order 1 of target trace
         i = np.argmax(trace1[row, :])
-        #tr = trace1[row, i-20:i+41]
-        tr = trace1[row, i-20:i+41]
-        w = tr/np.sum(tr**2)
+        tr = trace1[row, i - low_lim_col:i + high_lim_col]

```

Comment:
I would suggest using more explicit variable names (e.g. `trace` instead of `tr`), as variable names with one or two characters are sometimes hard to read.  I think PEP8 technically outlaws single-character variable names.